README.md

```diff
@@ -29,14 +29,14 @@ If the `ASSIGNMENT` environment variable is set, only _that_ exercise is tested.
 For example, if you only want to test the `two-fer.example.ts` for `two-fer`, you may, depending on your environment, use:
 
 ```shell
-ASSIGNMENT=two-fer yarn babel-node scripts/test
+ASSIGNMENT=practice/two-fer yarn babel-node scripts/test
 ```
 
 > Running on Windows? Depending on your shell, environment variables are set differently.
 > You can use `cross-env` to normalize this. The following should work across environments:
 >
 > ```bash
-> yarn cross-env ASSIGNMENT=two-fer babel-node scripts/test
+> yarn cross-env ASSIGNMENT=practice/two-fer babel-node scripts/test
```

Comment:
This being a windows example, should it say `practice\two-fer` instead? I don't know much about windows...

Reply:
No, we normalise slashes inside the script! 🗡️